mappings/net/minecraft/client/util/SelectionManager.mapping

```diff
@@ -6,38 +6,63 @@ CLASS net/minecraft/class_3728 net/minecraft/client/util/SelectionManager
 	FIELD field_24257 stringGetter Ljava/util/function/Supplier;
 	FIELD field_24258 stringSetter Ljava/util/function/Consumer;
 	FIELD field_24259 stringFilter Ljava/util/function/Predicate;
+	METHOD <init> (Ljava/util/function/Supplier;Ljava/util/function/Consumer;Ljava/util/function/Supplier;Ljava/util/function/Consumer;Ljava/util/function/Predicate;)V
+		ARG 1 stringGetter
+		ARG 2 stringSetter
+		ARG 3 clipboardGetter
+		ARG 4 clipboardSetter
+		ARG 5 stringFilter
 	METHOD method_16197 insert (Ljava/lang/String;)V
+		ARG 1 string
 	METHOD method_16198 deleteSelectedText (Ljava/lang/String;)Ljava/lang/String;
+		ARG 1 string
 	METHOD method_16199 insert (C)Z
 		ARG 1 c
 	METHOD method_16200 getSelectedText (Ljava/lang/String;)Ljava/lang/String;
+		ARG 1 string
 	METHOD method_16201 getSelectionStart ()I
 	METHOD method_16202 handleSpecialKey (I)Z
 		ARG 1 keyCode
 	METHOD method_16203 getSelectionEnd ()I
-	METHOD method_16204 moveCaretToEnd ()V
+	METHOD method_16204 moveCursorToEnd ()V
```

Comment:
How does this one relate to the version that take a boolean?

Reply:
Turns out the method was improperly named, it actually moves the cursor to the start, fixed